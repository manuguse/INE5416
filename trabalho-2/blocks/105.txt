1 5 8 11 11 11 20 20 23 28 28 28
1 1 8 8 8 12 12 23 23 29 29 28
1 6 6 12 12 12 19 19 19 29 33 33
1 6 2 12 12 19 19 19 24 24 33 33
1 2 2 2 16 14 19 24 24 30 30 33
2 2 2 13 14 14 21 21 24 26 30 33
3 7 7 14 14 14 21 21 25 26 30 30
3 7 9 15 15 14 21 21 26 26 30 30
3 4 9 15 15 15 15 22 22 22 31 31
3 4 4 10 15 17 22 22 22 31 31 31
4 4 10 10 17 17 22 18 27 27 27 31
4 4 10 10 18 18 18 18 18 32 32 32