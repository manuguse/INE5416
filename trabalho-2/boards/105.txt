- - - - - 3 - 1 3 - - -
5 - - 2 - - - - - 3 - 3
- 3 1 6 - - 4 7 - - - -
- - 6 5 4 2 - - - 1 - -
- - - 4 - - 1 - - 7 - 4
3 - - - 4 - 5 - - - 5 1
- - - - - 5 - 3 - - 4 -
- 1 - - 6 - - - 2 - - 1
- 7 - - - - 3 - 6 7 - 6
- - 3 - 4 - - - 3 - - 4
2 - - - 2 - - - 1 - - 2
- - 2 - 4 6 5 1 3 - 2 3