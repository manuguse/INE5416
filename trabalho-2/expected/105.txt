6 1 4 1 2 3 2 1 3 1 2 4
5 2 3 2 1 2 3 2 1 3 1 3
4 3 1 6 7 1 4 7 5 2 3 6
3 2 6 5 4 2 3 6 4 1 2 5
1 7 5 4 1 7 1 5 3 7 6 4
3 1 2 1 4 6 5 6 2 4 5 1
4 2 3 2 1 5 4 3 1 3 4 3
3 1 2 7 6 3 2 1 2 1 2 1
2 7 1 2 5 1 3 2 6 7 3 6
1 6 3 5 4 3 5 1 3 5 1 4
2 5 4 3 2 1 4 2 1 2 3 2
1 4 2 1 4 6 5 1 3 1 2 3